content/accessibility/developer-toolkit.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Developer toolkit
+---
+
+A great way to find answers to specific accessibility questions is to do an online search for the component/issue in question + 'a11y', 'accessible', or 'accessibility', and scan for official websites like W3C ([https://w3.org](https://w3.org)). 
+
+**Internal resources**
```

Comment:
I feel like this list of internal resources should be at the end of the file.
I also think the Fluent bit needs more context, that it's MSFT, etc.